packages/template-chakra-storefront/config/default.js

```diff
@@ -20,29 +20,27 @@ module.exports = {
             siteId: 'RefArchGlobal'
         }
     },
-    mobify: {
```

Comment:
In addition to making it flatter, by removing the `mobify`, pwa-kit-runtime also now has access to the entire config (which is necessary for private slas client).

Reply:
without `mobify` key, let's move it down the bottom of the file since we are arranging them alphabetically. 

If I understand it correcrlt, these 3 config ssr* is for MRT and the rest of config is for the app to use? If so, how about we call it something else instead of removing to create a clear boundaries

Reply:
@alexvuong MRT (or runtime in this case) actually needs both the ssr stuffs and also commerceAPI. 

Ok, I'll sort them alphabetically like how it was before.